index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <div id="main">
+      <div id="title"><h1>XXXTENTACION</h1></div>
+      <div id="img-div">
+          <img id="image" src="XXXTentacion.jpg" alt="a picture of X"></img>
```

Comment:
The img tag has no closing tag, it is called a self-closing tag. Please remove the closing tag you added.